### PR TITLE
Bugfix/multi tenancy example sample teams

### PR DESCRIPTION
**Pull request by kcoleman731**
### What does this PR do?

Adds test IAM user to multi-tenancy example. 

### Motivation

Test user is needed for e2e CI of example. 

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [x] Yes, I have added a new example under [examples](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/ssp-eks-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/docs) for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [x] E2E Test successfully complete before merge?

### CODE REVIEW

**Review by askulkarni2 — APPROVED** [review]

File: examples/multi-tenancy-with-teams/main.tf [hunk 1]
```diff
@@ -141,14 +141,13 @@ module "aws-eks-accelerator-for-terraform" {
   }
 
   platform_teams = {
-    admin-team-1 = {
+    admin = {
       ## Users Example:
       # users = [
       #   "arn:aws:iam::<ACCOUNT_ID>:user/<USERNAME>",
       #   "arn:aws:iam::<ACCOUNT_ID>:role/<ROLE_NAME>"
       # ]
```
Comment (vara-bonthu) [inline examples/multi-tenancy-with-teams/main.tf:149]: Can we remove this commented code or explain why we commented?

File: examples/multi-tenancy-with-teams/main.tf [hunk 2]
```diff
@@ -180,6 +179,7 @@ module "aws-eks-accelerator-for-terraform" {
...
```
Comment (vara-bonthu) [inline examples/multi-tenancy-with-teams/main.tf:181]: Same as above

File: examples/multi-tenancy-with-teams/main.tf [hunk 3]
```diff
@@ -203,6 +203,7 @@ module "aws-eks-accelerator-for-terraform" {
...
```
Comment (vara-bonthu) [inline examples/multi-tenancy-with-teams/main.tf:205]: Same as above

File: .github/workflows/examples-tfplan-tests.yml [hunk 4]
```diff
@@ -1,12 +1,10 @@
 name: e2e-all-examples-parallel
```
Comment (vara-bonthu) [inline .github/workflows/examples-tfplan-tests.yml:1]: Arn't we using this workflow now [e2e-parallel-full.yml](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/.github/workflows/e2e-parallel-full.yml)?
Should we remove this [examples-tfplan-tests.yml](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/pull/390/files#diff-186b79d57acb8e30733d94bdd38cc5fc0f0b1f42bc34653a123354964d557730)?

**Review by vara-bonthu — CHANGES_REQUESTED** [review]
Few queries on this workflow

File: .github/workflows/examples-tfplan-tests.yml (hunk 4 above)
Comment (Zvikan) [reply .github/workflows/examples-tfplan-tests.yml:1]: I believe the team wanted to keep TF plan only workflow for now.
We can think about having it with the new e2e-parallel, with some flag to say "Run only until e2e plan", so we can reduce the number of workflows, wdyt?

Comment (vara-bonthu) [reply .github/workflows/examples-tfplan-tests.yml:1]: fine with me

File: examples/multi-tenancy-with-teams/main.tf (hunk 1 above)
Comment (vara-bonthu) [reply examples/multi-tenancy-with-teams/main.tf:149]: I have been asked by few consultants on what this commented section is about. Do we need to enable etc.? May be we can add some wording to support that

**Review by askulkarni2 — APPROVED** [review]

**Review by Zvikan — APPROVED** [review]

**Pull request merged**